README.md

```diff
@@ -97,6 +97,18 @@ cargo +nightly fuzz run roundtrip_mime -- -max_len=10240
 cargo +nightly fuzz run roundtrip_random_config -- -max_len=10240
 ```
 
+no_std support
+---
+
+This crate has a feature flag `std` that is enabled by default.
+
+To use in a *no_std* environment, you can disable default features. To
+use the full API, enable feature `alloc` and setup an allocator.
+
+In environments without an allocator, the API provides only the
+`encode_config_slice()` and `decode_config_slice()` functions (which
+do not use the dynamically-sized `Vec` type) and the
+`strip_whitespace` config must be disabled.
```

Comment:
This unpleasant interplay of features is yet more evidence to me that line wrapping (and its dual, whitespace stripping) are a mistake to include, at least at this level. Maybe a `base64::ext` should contain helper functions to do that stuff after the fact. This would also simplify #56.